chevah/compat/tests/normal/test_capabilities.py

```diff
@@ -142,25 +124,75 @@ def test_openProcess_success(self):
                 process_token):
             self.assertIsNotNone(process_token)
 
+    def test_isPrivilegeEnabled_absent(self):
+        """
+        Returns False for a privilege which is not present.
+        """
+        # By default SE_RELABEL_NAME should not be available to test
+        # accounts.
+        privilege = win32security.SE_RELABEL_NAME
+        self.assertEqual(
+            u'absent', self.capabilities._getPrivilegeState(privilege))
+
+        self.assertFalse(self.capabilities._isPrivilegeEnabled(privilege))
+
+    def test_elevatePrivileges_invalid_privilege(self):
+        """
+        It raises an exception when an invalid privilege name is requested.
+        """
+        with self.assertRaises(AdjustPrivilegeException):
+            with (self.capabilities._elevatePrivileges(
+                    win32security.SE_IMPERSONATE_NAME,
+                    'no-such-privilege-name',
+                    )):
+                # pragma: no cover
+                pass
+
+    def test_get_home_folder(self):
+        """
+        On Windows, only Windows 2008 and Windows 7 can get home folder path.
+        """
+        # The Windows test is handled in elevated module.
+        pass
```

Comment:
I don't know, that's what the test said originally. I will look into what is going on.